src/cms/widgets/LocalizedString/control.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import * as R from 'ramda'
+import {
+  getTranslation,
+  uppendTranslation,
+  TranslationCollection,
+} from '../../i18n'
+import { Locale } from '../../i18n-locales'
```

Comment:
```suggestion
import { Locale } from '../i18n-config'
```

Reply:
I think we should import Locale from'../18n-config' everywhere and never import the library locales directly

Reply:
but maybe it's less coupled the way you have it

Reply:
(not a veto)

Reply:
I separated the lib `i18n-lib.(test.)ts` from the user interface `i18n.ts`. ONLY import from the later.